server/app/api/controllers/project_allocation.js

```diff
@@ -84,14 +86,15 @@ module.exports = {
   delete: function (req, res, next) {
     Project_Allocation_Model.findOneAndUpdate({ _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName
       },
-      function (err, userInfo) {
+      function (err, info) {
```

Comment:
info is not being used

Reply:
Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.